src/sentry/web/frontend/debug/debug_organization_integration_request.py

```diff
@@ -0,0 +1,30 @@
+from django.views.generic import View
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.models import Organization, OrganizationMember, User
+from sentry.notifications.notifications.organization_request.integration_request import (
+    IntegrationRequestNotification,
+)
+
+from .mail import render_preview_email_for_notification
+
+
+class DebugOrganizationIntegrationRequestEmailView(View):
+    def get(self, request: Request) -> Response:
+        org = Organization(id=1, slug="default", name="Default")
+        requester = User(name="Rick Swan")
+        recipient = User(name="James Bond")
```

Comment:
Should we check that this user has the ability of approving a Slack installation?
Since this is a debug view, how does the real code check that the recipient has this permission?